website/pages/blog/rebranding-in-open-source.mdx

```diff
@@ -0,0 +1,181 @@
+
+## The Guild today
+
+The Guild are a group of developers, creating open source, developer experience and infrastructure
+tools for managing data and APIs.
+
+We’ve managed to create industry leading tools, that are fully open source, while still being very
+profitable with solid financials that makes us resilient for the future.
+
+The whole structure of the group was to make sure we can create tools everyone can rely on for the
+very long term.
+
+We’ve also managed to become a really interesting place for ambitious developers to work in - all of
+our libraries are under developer’s names and not under The Guild’s org, in order to give the
+developers the fully power and credit they deserve.
+
+Building on all of these foundations together, we believe we’ve managed to create a strong ground
+for building quality software and we want to inspire others to create long term, quality software
+instead of short term, trend and marketing based software foundations.
+
+We’ve had a unique vision and a way to achieve it and we’ve manage to be successful at it.
+
+At this stage, we feel that more people should be aware of what we do.
+
+Some as users of our tools and some as other open source developers and solution providers, to share
+our tools and learnings with the wider community.
+
+We never done any marketing and we think this is a good time to start - At this stage, marketing to
+us is a force for bringing awareness to ideas we believe in
+
+## Where we want to be, our goals from the process
+
+As we go through the process, we have to ask ourselves what do we actually want to get.
+
+Our current goals as we see them today:
+
+- **A clear offering** - We offer many products, spread across many places across the stack. It is
+  currently very hard for someone new to The Guild’s ecosystem to understand what each tool does and
+  how it could benefit them
+- A philosophy of development - We’ve managed to create a philosophy around how we work, in order to
+  create sustainable, reliable, quality open source infrastructure. We want to share these ideas
+  with our users and other open source developers
+- **Hiring** - We want to give ambitious open source developers a career path that they are not
+  currently getting in other companies. Maximizing their individual potential is the core reason of
+  why we’ve created The Guild in the first place. We want to let developers know that The Guild is
+  more than a GraphQL provider and it’s also more than an API provider — it’s the open source
+  company, structured around the most solid solutions in open source that could be relied upon by
+  users and tool builders
+- **More exposure** - More developers to be aware of the available products we offer and why they
+  are the best in the industry (if they are not, we want to understand why as part of that process,
+  we want to share that as well in public). That includes the GraphQL ecosystem but also the wider
+  API ecosystem
```

Comment:
```suggestion
- **Clear offering:** We offer many products, spread across many places across the stack. It is
  currently very hard for someone new to The Guild’s ecosystem to understand what each tool does and
  how it could benefit them
- **Philosophy of development:** We’ve managed to create a philosophy around how we work, in order to
  create sustainable, reliable, quality open source infrastructure. We want to share these ideas
  with our users and other open source developers
- **Hiring:** We want to give ambitious open source developers a career path that they are not
  currently getting in other companies. Maximizing their individual potential is the core reason of
  why we’ve created The Guild in the first place. We want to let developers know that The Guild is
  more than a GraphQL provider and it’s also more than an API provider — it’s the open source
  company, structured around the most solid solutions in open source that could be relied upon by
  users and tool builders
- **More exposure:** More developers to be aware of the available products we offer and why they
  are the best in the industry (if they are not, we want to understand why as part of that process,
  we want to share that as well in public). That includes the GraphQL ecosystem but also the wider
  API ecosystem
```